src/single-spa-layout-main.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./single-spa-layout";
+export { WithLoadFunction } from "./constructApplications";
```

Comment:
I created this file so that we can export types, not just javascript exports, from the typescript definition for single-spa-layout.